### DISCUSSION TITLE
doris 1.2.1 query iceberg. it's mins level. need help to analyze  求助

### DISCUSSION THREAD

**Discussion by bossjie** [apache/doris#26311] [A - General / Q&A]
FE 3个节点, 16vcpu.64GBmem.80GBssd.2000eph
BE 4个节点, 16vcpu.64GBmem.664GBssd.2000eph
在master节点上 用mysql client, 创建catalog.
能顺利读取iceberg里的database和对应tables.
执行以下很小的表(dt是partition key).   第一次需要3 分多钟 ,第二次 第三次 会减少到几十秒. 
doris集群到iceberg集群都是内网(万兆网卡). 延迟0.x 毫秒
查询时 我也看了机器 cpu/mem, 都没飙高
也尝试了iceberg里的其他database的table,也是巨慢.    
查看了fe.log 也没error见着.   
select count(*) from pda.npsjoincallanalyzeaggregatehive where dt>='2023-09-01';  //dt为partition key
+----------+
| count(*) |
+----------+
|   128522 |
+----------+
1 row in set (17.21 sec)

如果可以的话, 十分希望加vx聊.   1051271623

感谢

**Comment by LemonLiTree** [comment]
try upgrading to 2.0.2

**Reply by bossjie** [reply]
@LemonLiTree  thanks for replying my ask.
https://www.infoq.cn/article/8su754vfukpw8gwjf24v
this blog says the speed to query iceberg is 3~10 faster than trino(presto). 
and it's using 1.2.x as I believe. I believe it's not release version related and big companies keep using Doris, they definitely have a good performance when querying iceberg using old version.   
Before upgrading version to 2.0.2, I'd like to learn what's happened in my scenerio. Thanks.
